### ISSUE TITLE
Validator failing when both "graphics-document" and "document" specified in SVG role

### ISSUE THREAD

**Issue by KDean-Dolphin** [validator/validator#1987]
### Summary

The validator is failing on an SVG tag like this:

```xml
<svg id="export-svg" width="100%" xmlns="http://www.w3.org/2000/svg" class="flowchart" style="max-width: 348.390625px; background: rgb(255, 255, 255);" viewBox="0 0 348.390625 491.765625" role="graphics-document document" aria-roledescription="flowchart-v2">...</svg>
```

The problem is the "role" attribute. The message coming back is:

```text
error: Bad value “graphics-document document” for attribute “role” on element “svg”.
```

The "graphics-document" role is specified in [WAI-ARIA Graphics Module](https://www.w3.org/TR/graphics-aria-1.0/#role_definitions), and the "role" attribute allows for multiple roles to be specified. You can see an example in the WAI-ARIA document itself in the linked section.

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
<svg id="export-svg" width="100%" xmlns="http://www.w3.org/2000/svg" class="flowchart" style="max-width: 348.390625px; background: rgb(255, 255, 255);" viewBox="0 0 348.390625 491.765625" role="graphics-document document" aria-roledescription="flowchart-v2">...</svg>
```

### Checker error messages

```shell
error: Bad value “graphics-document document” for attribute “role” on element “svg”.
```

### Spec links/citations

https://www.w3.org/TR/graphics-aria-1.0/#role_definitions

### Steps to reproduce

1.
2.
3.

### Expected behavior

SVG passes cleanly.

### Actual behavior

Error message.

**Comment by sideshowbarker** [comment]
Thanks for catching this and reporting it. The problem was that we didn’t recognize the `graphics-document` role value, at all.

That’s now fixed, and deployed to https://validator.w3.org/nu/

However, `role="graphics-document document"` is still going to cause an error — though now, a different one:

>  _Discarding superfluous token `document` from value of attribute `role`. Browsers only process the first token found that is a defined ARIA non-abstract role._

That error is completely intentional. We don’t emit that for just the SVG case, but instead for _any_ value of a `role` attribute which contains multiple non-abstract role names.

But if you don’t think we should be emitting any error for that case, please file a separate new issue.
